python/destinations/ConfluentKafka/main.py

```diff
@@ -29,15 +29,8 @@
 
     # Callback called for each incoming stream
     def read_stream(input_stream: StreamReader):
-
```

Comment:
Why we have HiveMQ stuff in Confluente Kafka file?

Reply:
No idea. based on Peter's comment: https://github.com/quixai/quix-library/pull/223#discussion_r919006233